### PR TITLE
Added README.md deliverable requirement and link to rubric pdf

### PR DESCRIPTION
**Pull request by lmcm18**
I added the readme deliverable requirement to the deliverables section and a link to the rubric, which right now points to where the pdf will be in the master branch, in the requirements section. @LoreDirick

### CODE REVIEW

**Review by LoreDirick — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,69 +31,17 @@ You'll clean, explore, and model this dataset with a multivariate linear regress
 
 ## The Deliverables
 
-There will be four  deliverables for this project:
+For online students, there will be five deliverables for this project:
```
Comment (DavidBraslow) [inline README.md]: Can we use the same language from the Blog repo? "Note: On-campus students may have different requirements, please speak with your instructor."

File: README.md [hunk 2]
```diff
@@ -31,69 +31,17 @@ You'll clean, explore, and model this dataset with a multivariate linear regress
...
-* You won't be put on the spot without support
-* There's nothing you can do to instantly fail or blow it
-
-**Please note: We need to receive the URL of your repository at least 24 hours before and please have the project finished at least 3 hours before your review so we can look at your materials in advance.**
-
-
-## Requirements
-
-This section outlines the rubric we'll use to evaluate your project.
+For on-campus students, please talk to your instructor for a list of required project deliverables.
```
Comment (DavidBraslow) [inline README.md]: Can we use the same language here as well? "Note: On-campus students may have different requirements, please speak with your instructor."

File: README.md [hunk 3]
```diff
@@ -132,10 +80,62 @@ Your presentation should:
...
 **_Based on the results of your models, your presentation should discuss at least two concrete features that highly influence housing prices._**
 
-### 3. Blog Post
+### Blog Post Must-Haves
 
 Refer back to the [Blogging Guidelines](https://github.com/learn-co-curriculum/dsc-welcome-blogging) for the technical requirements and blog ideas.
 
+
+## The Process
+
```
Comment (DavidBraslow) [inline README.md:90]: This also will differ for in-person: "Note: On-campus students may have different processes, please speak with your instructor."

File: README.md [hunk 4]
```diff
@@ -146,6 +146,10 @@ _Reminder: Make sure to also add and commit a pdf of your non-technical presenta
 3. **Record Walkthrough:** Include a link to your video walkthrough.
 
  Hit "I'm done" to wrap it up. You will receive an email in order to schedule your review with your instructor.
+ 
+ 
+## Grading Rubric
+Online students can find a PDF of the grading rubric for the project [here](https://github.com/learn-co-curriculum/dsc-v2-mod1-final-project/blob/master/module1_project_rubric.pdf). On-campus students should contact their instructor for more information.
```
Comment (DavidBraslow) [inline README.md:156]: Here, too: "Note: On-campus students may have different review processes, please speak with your instructor."

File: README.md [hunk 5]
```diff
@@ -132,10 +80,62 @@ Your presentation should:
...
+### 1. Getting Started
+
+Please start by reviewing this document. If you have any questions, please ask them in Slack ASAP so (a) we can answer the questions and (b) so we can update this repository to make it clearer.
+
+Be sure to let the instructor team know when you’ve started working on a project, either by reaching out over Slack or, if you are in a full-time or part-time cohort, by connecting with your Cohort Lead in your weekly 1:1. If you’re not sure who to reach out to, post in the #online-ds-sp-000 channel in Slack.
+
+Once you're done with the first 8 sections, please start on the project. Do that by forking this repository, cloning it locally, and working in the student.ipynb file. Make sure to also add and commit a pdf of your presentation to the repository with a file name of `presentation.pdf`.
+
+### 2. The Project Review
+
```
Comment (DavidBraslow) [inline README.md:100]: Here, too: "Note: On-campus students may have different review processes, please speak with your instructor."

**Review by DavidBraslow — COMMENTED** [review]
Maybe we can just put a big disclaimer at the top: "Note: On-campus students may have different project requirements and processes. Your teachers should communicate these to you during your course, and you should consult with them if you have any questions."

**Review by ghost — APPROVED** [review]

**Review by LoreDirick — APPROVED** [review]

**Comment by LoreDirick** [comment]
updated using David's feedback

**Pull request merged**